### ISSUE TITLE
id="give-amount" output multiple times on the page

### ISSUE THREAD

**Issue by mathetos** [WordImpress/Give#326]
## A11Y Report Notes
http://b8d.3e8.myftpupload.com/
**DONATION FORM WIDGET**

* Error /  priority 65%
* Duplicate ID found
* (line: 138, 3)

`<input id="give-amount" class="give-amount-hidden" type="hidden" name="give-amount" value="5.00" required="" style="">`

**Web Content Accessibility Guidelines (WCAG) 2.0, Level A: 4.1.1 Parsing**

An ID attribute value has been used more than once. In cases where the ID is used to reference a UI control this is likely to cause issues for users of assistive technologies

## Possible Solutions
* Dynamically add the Form ID to the end
* Determine whether input actually requires an ID at all, perhaps not

**Comment by mathetos** [comment]
@TheGeniusWM What's your take on this? Do input even need an ID at all? Are there accessibility issues with inputs having ID's? basically, maybe we can just remove the ID safely to avoid this problem and not adversely affect the styles or functionality at all. Let me know your thoughts.

**Comment by TheGeniusWM** [comment]
Let me dig into this one before you work on it. I will comment with a definitive answer shortly.

**Comment by TheGeniusWM** [comment]
Ok so it is recommended that all form inputs have proper labels, and that labels that exist on the same page need to be unique.
For someone using a screenreader, inputs with the same ID can get combined and inputs could potentially be missed,
A hidden label would be a valid fix. As long as the assistive technology can read it in the code we are good.

**Comment by mathetos** [comment]
This should be resolved with PR #358
